jdaviz/configs/default/plugins/plot_options/tests/test_plot_options.py

```diff
@@ -172,3 +177,63 @@ def test_user_api(cubeviz_helper, spectrum1d_cube):
     # toggle contour (which has a spinner implemented)
     po.contour_visible = True
     assert po._obj.contour_spinner is False
+
+
+@pytest.mark.filterwarnings('ignore')
```

Comment:
What warnings are you ignoring here? Is it possible to localize it? Blanket ignore like this can easily let bugs through if we are not careful.